examples/bitmask_compression.ipynb

```diff
@@ -77,14 +133,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "The example layer model.layers.0.self_attn.q_proj.weight has sparsity 0.50%\n"
+      "The example layer model.layers.0.self_attn.q_proj.weight has sparsity 50.00%\n"
```

Comment:
```suggestion
      "The example layer model.layers.0.self_attn.q_proj.weight has sparsity 50%\n"
```
nit: no need for so many sig figs :)

Reply:
updated the print statement to drop the decimals entirely